src/py-opentimelineio/CMakeLists.txt

```diff
@@ -4,18 +4,20 @@
 add_subdirectory(opentime-bindings)
 add_subdirectory(opentimelineio-bindings)
 
-# Install pure-python OTIO packages to match PyPI wheel structure
-install(DIRECTORY "${PROJECT_SOURCE_DIR}/src/py-opentimelineio/opentimelineio/"
-    DESTINATION "${OTIO_RESOLVED_PYTHON_INSTALL_DIR}/opentimelineio")
+if(NOT OTIO_PYTHON_INSTALL_DIR)
```

Comment:
I feel like this is fragile. I was thinking of adding a new variable like `CALLED_FROM_SETUPTOOLS` or something like that which would be more reliable. Opinions?

Reply:
maybe `OTIO_INVOKED_FROM_SETUPTOOLS` to keep with the `OTIO_` pattern for outside visibility?

Reply:
This seems to be the core bug here, so flagging the case explicitly as you suggest seems like the right answer, and maybe that can help clarify the logic I was asking about above.

Reply:
ok, now that the above is resolved, maybe this clause is fine now?

Reply:
I still feel like it's fragile. It relies on logics while I think a dedicated variable would be better. Also, what do you think of `OTIO_PYTHON_MODULES_INSTALL` instead of `OTIO_INVOKED_FROM_SETUPTOOLS`. It would be slightly more generic.